applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -130,8 +130,8 @@ static void print_codec(const struct audio_codec_info *codec)
 {
 	if (codec->id == BT_HCI_CODING_FORMAT_LC3) {
 		LOG_INF("Codec config for LC3:");
-		LOG_INF("\tFrequency: %d Hz", codec->frequency);
-		LOG_INF("\tFrame Duration: %d us", codec->frame_duration_us);
+		LOG_INF("\tFrequency: %d Hz", le_audio_freq_get_hz(codec));
```

Comment:
This is already calculated in the function below, so should just need to print here, same with the with the one below

Reply:
So this system will always run get_codec_info() before print_codec() and not directly printing the configuration?

Reply:
They are using two different structs, this print function is using our own `audio_codec_info` that is populated in the function `get_codec_info()`